AlphaWallet/Accounts/ViewModels/WalletSummaryViewModel.swift

```diff
@@ -77,13 +77,12 @@ extension WalletSummaryViewModel {
 
             let changePercentage: String = {
                 guard let changePercentage = summary?.changePercentage else { return "-" }
-                let formatter = NumberFormatter.priceChange(currency: changePercentage.currency)
```

Comment:
Maybe replace the `let` variable with `NumberFormatter.percent` instead? Few lines and better